### PR TITLE
Update notebooks

### LINKED ISSUE #106: Updating the existing example notebooks
We have to update and fix the existing notebooks (some of them have bugs) and create some new ones (for example for the `SRRes` architecture).

### PR DESCRIPTION
**Pull request by FeGeyer**
Update of the example notebooks to the latest version. Fixes #106

### CODE REVIEW

File: radionets/evaluation/plotting.py [hunk 1]
```diff
@@ -288,8 +288,8 @@ def visualize_with_fourier_diff(
     real_truth, imag_truth = img_truth[0], img_truth[1]
 
     if amp_phase:
-        real_pred = 10 ** (10 * np.array(real_pred, dtype="float128") - 10) - 1e-10
-        real_truth = 10 ** (10 * np.array(real_truth, dtype="float128") - 10) - 1e-10
```
Comment (Kevin2) [inline radionets/evaluation/plotting.py:292]: why is this no problem anymore?

**Comment by Kevin2** [comment]
in notebook 01:
mix of all is in principal the same example as point sources with extended sources enabled as shown before, right?
the log scale image in mix of all does not look very nice in my opinion 
should we drop this?

File: examples/01_dataset_simulation.ipynb [hunk 2]
```diff
@@ -46,7 +46,7 @@
     {
```
Comment (Kevin2) [inline examples/01_dataset_simulation.ipynb:46]: <p>is here still the chess board pattern for the imag image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radionets-project/radionets/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (Kevin2) [inline examples/01_dataset_simulation.ipynb:46]: <p>is tex only failing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radionets-project/radionets/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

File: examples/03_model_evaluation.ipynb [hunk 3]
```diff
@@ -130,13 +130,13 @@
     {
```
Comment (Kevin2) [inline examples/03_model_evaluation.ipynb]: <p>maybe add comment, that training time is not enough for a reasonable prediction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radionets-project/radionets/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Comment by Kevin2** [comment]
nice work! :)

File: radionets/evaluation/plotting.py (hunk 1 above)
Comment (FeGeyer) [reply radionets/evaluation/plotting.py:292]: I don't remember what the exact problem was. I tested some things after removing this and everything seemed fine. This casting produces a warning which I wanted to remove. MAybe you can remind me, what the problem was.

**Comment by FeGeyer** [comment]
> in notebook 01: mix of all is in principal the same example as point sources with extended sources enabled as shown before, right? the log scale image in mix of all does not look very nice in my opinion should we drop this?

Yeah, this is the same example because of #108. So if we fix this, we can show different examples.

File: radionets/evaluation/plotting.py (hunk 1 above)
Comment (Kevin2) [reply radionets/evaluation/plotting.py]: I think, we had some problems with the evaluation, when we did fast implementations during the review process of the paper. I cannot remember the exact problem. We should keep it in mind, but should be fine for the moment.

File: examples/01_dataset_simulation.ipynb (hunk 2 above)
Comment (Kevin2) [reply examples/01_dataset_simulation.ipynb:46]: maybe my plots were just outdated

Comment (Kevin2) [reply examples/01_dataset_simulation.ipynb:46]: <p>know it is working for me aswell</p>

Comment (Kevin2) [reply examples/01_dataset_simulation.ipynb:46]: <p>or I got confused because of the white noise</p>

**Comment by Kevin2** [comment]
<p>Okay, we will fix #108 soon </p>

**Review by Kevin2 — APPROVED** [review]

**Pull request merged**